mne/viz/evoked.py

```diff
@@ -1198,3 +1199,398 @@ def plot_evoked_joint(evoked, times="peaks", title='', picks=None,
+        evokeds = dict(Evoked=evokeds)
+    else:
+        if isinstance(evokeds, dict):
+            conditions = sorted(list(evokeds.keys()))
+        else:
+            evokeds = dict((str(ii + 1), evoked)
+                           for ii, evoked in enumerate(evokeds))
+            conditions = sorted(list(evokeds.keys()))
+
+    # get and set a few limits and variables (times, channels, units)
+    example = (evokeds[conditions[0]]
+               if isinstance(evokeds[conditions[0]], Evoked)
+               else evokeds[conditions[0]][0])
+    if not isinstance(example, Evoked):
+            raise ValueError("evokeds must be an `mne.Evoked` "
+                             "or of a collection of `mne.Evoked`s")
+    times = example.times
+    tmin, tmax = times[0], times[-1]
+
+    # deal with picks: infer indices and names
+    if picks == 'gfp':
+        if ymin is None:
+            ymin = 0
+        gfp_picks = pick_types(example.info, eeg=True, meg=True, seeg=True,
+                               ecog=True)
+        ch_types = list(set([channel_type(example.info, pick)
+                             for pick in gfp_picks]))
+        if len(ch_types) > 1:
+            raise ValueError("Can't plot GFP with multiple channel types.")
+        ch_type = ch_types[0]
+        ch_names = ['Global Field Power']
+    else:
+        if isinstance(picks, int):
+            picks = [picks]
+        elif isinstance(picks[0], int):
+            pass
+        else:
+            raise ValueError("`picks` must be int, a list of int, "
+                             "or `gfp`, not " + str(type(picks)))
+        ch_names = [example.ch_names[pick] for pick in picks]
+        ch_type = channel_type(example.info, picks[0])
+        ch_types = list(set(channel_type(example.info, pick_)
+                        for pick_ in picks))
+        if len(ch_types) > 1:
+            raise ValueError("More than 1 channel type selected by `picks`.")
+
+    scaling = _handle_default("scalings")[ch_type]
+
+    if ch_type == 'grad' and picks is not 'gfp':  # deal with grad pairs
+        from ..channels.layout import _merge_grad_data, _pair_grad_sensors
+        picked_chans = []
+        pairpicks = _pair_grad_sensors(example.info, topomap_coords=False)
+        for ii in np.arange(0, len(pairpicks), 2):
+            first, second = pairpicks[ii], pairpicks[ii + 1]
+            if first in picks or second in picks:
+                picked_chans.append(first)
+                picked_chans.append(second)
+        picks = list(sorted(set(picked_chans)))
+        ch_names = [example.ch_names[pick] for pick in picks]
+        if ymin is None:  # 'grad' is plotted as all-positive
+            ymin = 0
+
+    # deal with dict/list of lists, and calculate the CI
+    if not isinstance(ci, np.float):
+        msg = '`ci` must be float, got {0} instead.'
+        raise TypeError(msg.format(type(ci)))
+
+    if not all([isinstance(evoked_, Evoked) for evoked_ in evokeds.values()]):
```

Comment:
Ah wait, I misread.

No, this is for when the input is a list of lists of evokeds, or a dict of lists of evokeds, in which case here the CI is computed and later the Grand Mean.